hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/schemas/InitialModServiceAdminSchema.java

```diff
@@ -61,15 +63,16 @@ public void migrate(@NonNull final MigrationContext ctx) {
         final var upgradeFileHashKeyState =
                 ctx.newStates().<ProtoBytes>getSingleton(FreezeServiceImpl.UPGRADE_FILE_HASH_KEY);
         final var freezeTimeKeyState = ctx.newStates().<Timestamp>getSingleton(FreezeServiceImpl.FREEZE_TIME_KEY);
-        final var lastFrozenTimeKeyState =
-                ctx.newStates().<Timestamp>getSingleton(FreezeServiceImpl.LAST_FROZEN_TIME_KEY);
 
-        if (isGenesis) {
+        if (isGenesis || mnc != null) {
             upgradeFileHashKeyState.put(ProtoBytes.DEFAULT);
             freezeTimeKeyState.put(Timestamp.DEFAULT);
-            lastFrozenTimeKeyState.put(Timestamp.DEFAULT);
         }
 
         log.info("BBM: no migration actions necessary for admin service");
```

Comment:
Two things here:
1. Null the reference to the network state, so it can be released (critical). I.e. set `mnc = null;` 
2. We'll probably want to change this logging statement :)

Reply:
Yes, thanks @mhess-swl . Fixed